### PR TITLE
Fix config option descriptions

### PR DESCRIPTION
**Pull request by nkakouros**
##### SUMMARY
Simple text edits.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME
Configuration

##### ANSIBLE VERSION

```
ansible 2.5.0
  config file = None
  configured module search path = ['/home/nikos/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.6/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.6.4 (default, Jan  5 2018, 02:35:40) [GCC 7.2.1 20171224]
```

### CODE REVIEW

**Comment by mkrizek** [comment]
cc @bcoca @gundalow

File: lib/ansible/config/base.yml [hunk 1]
```diff
@@ -842,7 +842,8 @@ DEFAULT_PRIVATE_KEY_FILE:
 DEFAULT_PRIVATE_ROLE_VARS:
   name: Private role variables
   default: False
-  description: ''
+  description:
+    - Makes role variables inaccessible from other roles (and the role that own the variables if replayed).
```
Comment (gundalow) [inline lib/ansible/config/base.yml]: Wonder if it would make sense to specify *why* this may be useful?

Comment (nkakouros) [reply lib/ansible/config/base.yml]: Perhaps like this? Is there something else worth mentioning?

**Comment by bcoca** [comment]
rebuild_merge

**Comment by gundalow** [comment]
rebuild_merge

**Comment by nkakouros** [comment]
Could you explain what these `rebuild_merge` do? Where can I learn more about these?

**Comment by bcoca** [comment]
@tterranigma it allows someone with commit privileges (me) to make bot rerun tests and merge if they pass. As for learning about these commands: https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md

**Pull request merged**